### PR TITLE
SO-3239: Add inboundRelationships expand option to concept search and get requests

### PR DESCRIPTION
**Pull request by zoliMolnar**

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/SnomedConceptConverter.java [hunk 1]
```diff
@@ -311,6 +312,33 @@ private void expandRelationships(List<SnomedConcept> results, final Set<String>
...
+	
+	private void expandInboundRelationships(List<SnomedConcept> results, final Set<String> conceptIds) {
+		if (!expand().containsKey(SnomedConcept.Expand.INBOUND_RELATIONSHIPS)) {
+			return;
+		}
+		
+		final Options expandOptions = expand().get(SnomedConcept.Expand.INBOUND_RELATIONSHIPS, Options.class);
+		final int relationshipSearchLimit = getLimit(expandOptions);
+		
+		final SnomedRelationships inboundRelationships = SnomedRequests.prepareSearchRelationship()
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/SnomedConceptConverter.java:327]: Please, add support for filtering inbound relationships by `typeId` and `sourceId`.

**Review by cmark — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/components/SnomedConceptApiTest.java [hunk 2]
```diff
@@ -66,7 +66,7 @@
  */
 public class SnomedConceptApiTest extends AbstractSnomedApiTest {
 
-	@Test
+	//@Test
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/components/SnomedConceptApiTest.java]: Please add these tests back to the test suite. Thank you!

**Review by cmark — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedConcept.java [hunk 3]
```diff
@@ -307,6 +317,10 @@ public void setRelationships(SnomedRelationships relationships) {
 		this.relationships = relationships;
 	}
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedConcept.java]: Javadoc is missing from `setInboundRelationships` method.

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedConcept.java [hunk 4]
```diff
@@ -204,6 +205,15 @@ public SnomedDescriptions getPreferredDescriptions() {
 	public SnomedRelationships getRelationships() {
 		return relationships;
 	}
+	
+	/**
+	 *
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedConcept.java]: Remove unnecessary spaces/newlines.

**Review by cmark — CHANGES_REQUESTED** [review]

**Review by cmark — APPROVED** [review]
LGTM!

**Comment by zoliMolnar** [comment]
Thank you!

**Pull request merged**